src/api/routers/user.py

```diff
@@ -22,6 +22,8 @@
 class IdentityResponse(BaseModel):
     uid: Optional[str]
     status: Optional[str]
+    submission_time: Optional[datetime]
+    verdict_time: Optional[datetime]
```

Comment:
Thought: when initially planning out the API integration, I considered having a separate endpoint for applicant-specific data rather than cluttering the global user identity. This would be helpful/cleaner if we had other user roles interacting with our site, but when discussing with directors earlier this week, most likely, only applicants and organizers (no mentors, volunteers, workshop hosts, sponsors, or judges) will need site access.